server/Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM eligibility_verification_client:latest
+FROM eligibility_verification_client:dev
```

Comment:
It's probably fine to just call it latest and not dev, because we'll never ship `latest`, we'll always ship the specific sha tag.

Reply:
This is the Dockerfile for our test server, which is not going to be part of the production deployment. It's built from the `dev` version of the image because of some debugging packages installed that are not installed in `latest`.